File: Arcade/TheCore/Javascript/replaceMiddle.js
function replaceMiddle(arr) {
    if (arr.length % 2 == 0)
        arr.splice(
            arr.length / 2 - 1, 2,
            arr[arr.length / 2] + arr[arr.length / 2 - 1]
        )
    
    return arr
}

File: Arcade/TheCore/Java/ReplaceMiddle.java
final class ReplaceMiddle {

    int[] replaceMiddle(int[] arr) {
        if (arr.length % 2 > 0) return arr;
        
        int sum = arr[arr.length / 2 - 1] + arr[arr.length / 2];
        
        arr[arr.length / 2 - 1] = sum;
        
        return IntStream.range(0, arr.length)
            .filter(i -> i != arr.length / 2)
            .map(i -> arr[i]).toArray();
    }
}